### ISSUE TITLE
[COMMENTS] Quotient groups and modular integers

### ISSUE THREAD

**Issue by jtiosue** [jtiosue/jtiosue.github.io#8]

**Comment by liandrea4** [comment]
The inline glossary is super useful! Now I know the definitions of complicated terms I'm not familiar with just by hovering over the word! You must be mart :)

**Comment by jtiosue** [comment]
Testing math $x^2$ and

$$\frac{x}{2}$$

**Comment by jtiosue** [comment]
> The inline glossary is super useful! Now I know the definitions of complicated terms I'm not familiar with just by hovering over the word! You must be mart :)

That's true! You should do it too!! I get confused on your [blog](https://liandrea4.github.io/)..
